cypress/e2e/integration/pages/fin.cy.js

```diff
@@ -1,9 +1,54 @@
+import {
+  FIN_EMAIL_INPUT,
+  FIN_EMAIL_SUBMIT_BUTTON,
+} from '../../../constants/elements-ids'
+import { clickSkipTutorialButton } from '../../../helpers/elements/buttons'
 import { visit } from '../../../helpers/interactions/visit'
+import { recursivelyFillSimulation } from '../../../helpers/simulation/recursivelyFillSimulation'
+import { skipRiddle } from '../../../helpers/simulation/skipRiddle'
 
 describe('The Fin page', () => {
   it('should redirect to the tutorial when no simulation exists', () => {
     visit('fin')
 
+    cy.wait(4000)
+
     cy.get('h1[data-cypress-id="tutoriel-title"]').should('be.visible')
   })
+
+  context(
```

Comment:
@paulsouche pas sûr de l'enchevêtrement de context et it ici pour respecter given, when, then ; preneur de retour

Reply:
Merci pour la structure du test, je reformule